slick/src/main/scala/slick/sql/SqlCapabilities.scala

```diff
@@ -16,6 +16,8 @@ object SqlCapabilities {
   val sequenceMax = Capability("sql.sequenceMax")
   /** Supports min value for sequences */
   val sequenceMin = Capability("sql.sequenceMin")
+  /** supports interpreting strings as VARCHAR. Some databases eg. HSQL do not do this. */
+  val stringIsVarchar = Capability("sql.stringIsVarchar")
```

Comment:
I read about this [here](https://github.com/slick/slick/blob/master/slick/src/main/scala/slick/jdbc/HsqldbProfile.scala#L76).
Hence I am using this capability. It might be useful to make the description more verbose. Should I do that?